be/src/column/nullable_column.cpp

```diff
@@ -136,7 +136,7 @@ bool NullableColumn::append_nulls(size_t count) {
     if (count == 0) {
         return true;
     }
-    _data_column->append_default(count);
+    _data_column->resize_uninitialized(_data_column->size() + count);
```

Comment:
Will `resize_uninitialized` destroy previous value?

Reply:
basically this is calling `std::vector::resize`, as per `https://cplusplus.com/reference/vector/vector/resize/`:

Reply:
OK.